class-instant-articles-post.php

```diff
@@ -737,12 +737,27 @@ public function add_ads_from_settings() {
 			$height = intval( $dimensions_match[2] );
 		}
 
+		$source_of_ad = isset( $settings_ads['ad_source'] ) ? $settings_ads['ad_source'] : 'none';
+
+		$ad = $this->get_ads_for_ad_source($source_of_ad, $width, $height);
```

Comment:
What do you think of a oneliner with documentation:

```
    $ad = apply_filters( 
        'instant_articles_loading_ads', 
        $this->get_ads_for_ad_source( $source_of_ad, $width, $height)
    );
    $header->addAd( $ad );
```

Mind the code formatting of spaces ( param ).
Also add the doc block for this filter.